src/schema/v2/gene.ts

```diff
@@ -33,12 +37,59 @@ const SUBJECT_MATTER_MATCHES = [
 
 const SUBJECT_MATTER_REGEX = new RegExp(SUBJECT_MATTER_MATCHES.join("|"), "i")
 
+const META_DESCRIPTION_OVERRIDES = {
+  "western-europe":
+    "Discover Western European artists from pre-history to present, and browse works by size, prize and medium.",
```

Comment:
typo **price**, and medium

Reply:
Ah, weird; these were copied out of Force :p